docs/area-owners.md

```diff
@@ -42,26 +41,25 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
 | area-ILTools-coreclr                           | @JulieLeeMSFT        | @BruceForstall @dotnet/jit-contrib                  |                                                                                                                                                                                                                                                                                           |
 | area-Infrastructure                            | @agocke              | @jeffschwMSFT @MichaelSimons                        |                                                                                                                                                                                                                                                                                           |
-| area-Infrastructure-coreclr                    | @agocke              | @jeffschwMSFT @trylek                               |                                                                                                                                                                                                                                                                                           |
+| area-Infrastructure-coreclr                    | @agocke              | @jeffschwMSFT                                       |                                                                                                                                                                                                                                                                                           |
 | area-Infrastructure-installer                  | @MichaelSimons       | @NikolaMilosavljevic                                |                                                                                                                                                                                                                                                                                           |
 | area-Infrastructure-libraries                  | @ericstj             | @dotnet/area-infrastructure-libraries               | Covers:<ul><li>Packaging</li><li>Build and test infra for libraries in dotnet/runtime repo</li><li>VS integration</li></ul><br/>                                                                                                                                                          |
 | area-Infrastructure-mono                       | @steveisok           | @directhex                                          |                                                                                                                                                                                                                                                                                           |
```

Comment:
@jeffschwMSFT @lewing @steveisok  who should this change to?

Reply:
I left Jo in that space as he has a lot of knowledge and still works in the repo.  I can remove him for now.  We discussed a need for some infrastructure triage to sort this out (which is pending)

Reply:
We can keep Jo listed as a consultant but not as owner / responsible party.